### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by marijanbrvar**
In this project, I use redux and provider to pass data to components.
- Use the concept of state management.
- Use store, actions, and reducers in React.
- Connect React and Redux.
- Add React Router and set two routes

### CODE REVIEW

File: src/components/AddNewBook.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        >
+          <input type="text" name="title" placeholder="Title" required />
+          <select name="category" required>
+            {category}
+          </select>
+
+          <button type="submit" className="submitBtn">Add</button>
+        </form>
+      </div>
+    </div>
```
Comment (DeliceLydia) [inline src/components/AddNewBook.js]: - [x] you should render a form with inputs for the `title` and the `author` of the book currently you have only the title please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @marijanbrvar, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddNewBook.js (hunk 1 above)
Comment (oxenprogrammer) [reply src/components/AddNewBook.js]: - [ ] Well done implementing the requested change.

**Review by oxenprogrammer — APPROVED** [review]
Hi @marijanbrvar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**